datasets/cloud_storage_geo_index/pipelines/_images/cloud_storage_geo_index_landsat_index_schema.json

```diff
@@ -0,0 +1,104 @@
+[
```

Comment:
I think these files should be inside the `run_csv_transform_kub` folder, because they're expected to be files inside the container.

Reply:
ok.  will do.

Reply:
done